pkg/dogstatsd/server.go

```diff
@@ -341,6 +347,39 @@ func NewServer(demultiplexer aggregator.Demultiplexer, serverless bool) (*Server
 		}
 	}
 
+	// Creating a new logger
+	cfg := seelogCfg.NewSeelogConfig(
+		"dogstatsd",
+		"info",
+		"common",
+		"",
+		"%Date %Time | [DOGSTATSD_STATS] | [%LEV] %Msg ",
+		false)
+
+	// Configuring the log file path
+	logFile := config.Datadog.GetString("dogstatsd_log_file")
```

Comment:
I wonder what happens if this is set to the path to the main `agent.log` file?

Reply:
Its a good thought, but we don't handle this mistake in the `jmx_log_file` for example, so maybe this kind of extra error-checking would be best left for a future PR. (would probably be a good "first-change" task for a new embed/intern)